.gitignore

```diff
@@ -2,3 +2,4 @@
 *.pyc
 .DS_Store
 _site
+.idea/
```

Comment:
Do you mind using a [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) for this? Otherwise people could start asking to include others IDE-specific files.